src/lib/CanvasAnnotationDisplay.js

```diff
@@ -104,9 +105,12 @@ export default class CanvasAnnotationDisplay {
 
   /** */
   fragmentContext() {
-    const fragment = this.resource.fragmentSelector;
+    let fragment = this.resource.fragmentSelector;
     fragment[0] += this.offset.x;
```

Comment:
I'm wondering now if perhaps we are calculating the offset incorrectly? Scaling the offsets before applying seem to be a useful transformation.

Reply:
So I'm thinking we might need to do the following:

1. Better calculate an offset based off of the scaling
2. Resize the fragment/svg based off of scaling